### DISCUSSION TITLE
Mustache Templates - Subflow Environment Variables

### DISCUSSION THREAD

**Discussion by snadboy** [zachowj/node-red-contrib-home-assistant-websocket#496] [Q&A]
Since mustache templates permit access to global and flow variables, is it possible to access an environment variable in a subflow in a similar fashion?

**Comment by zachowj** [comment]
1. replace a whole property value https://nodered.org/docs/user-guide/environment-variables
2. insert it into a string using JSONata, `"first second " & $env('varname') & "fourth"`
3. insert into the `msg` with a change node then access it using mustache templates or JSONata
